zstacklib/ansible/zstacklib.py

```
@@ -2324,10 +2329,373 @@ def install_release_on_host(is_rpm, host_info, host_post_info):
+            continue
+            ;;
+    esac
+
+    [ "$pid" = "$$" ] && continue
+    [ "$pid" = "$PPID" ] && continue
+    proc_pgid="$(ps -o pgid= -p "$pid" 2>/dev/null | tr -d ' ')"
+    [ -n "$pgid" ] && [ "$proc_pgid" = "$pgid" ] && continue
+    echo "$pid"
+done | sort -u
+"""
+    status, output = _run_rpmdb_repair_command(cmd, host_post_info,
```

Ruff (0.15.12)

[warning] Unpacked variable `status` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)